arch/arm64/configs/sun50iw1p1smp_linux_defconfig

```diff
@@ -137,15 +137,15 @@ CONFIG_RD_GZIP=y
 CONFIG_RD_BZIP2=y
 CONFIG_RD_LZMA=y
 CONFIG_RD_XZ=y
-# CONFIG_RD_LZO is not set
+CONFIG_RD_LZO=y
```

Comment:
This is a default which gets applied every time you use the default template. It is just the right value now.